src/pages/home/supportTheCauseSection.js

```diff
@@ -24,11 +24,21 @@ export default function SupportTheCauseSection() {
 				To help us grow these opportunities, please consider making a
 				tax-deductible donation to SpaceLab today.
 			</p>
-			<div id={"contact-us-button-container"}>
+			{/* <div id={"contact-us-button-container"}>
 				<NavigationButton
-					pathName={"/contact"}
-					buttonText={"CONTACT US"}
+					pathName={"/donate"}
+					buttonText={"Donate"}
+					href="https://www.paypal.com/donate/?hosted_button_id=PK9D4A3HEWV8C"
 				/>
+			</div> */}
+			<div id={"contact-us-button-container"}>
```

Comment:
This id no longer makes sense. This should be a donate button container. I assume css will need to be updated to match.